sample-files/make_ghost_test.py

```
@@ -0,0 +1,73 @@
+
+out = "ghost_test.pcap"
+wrpcap(out, pkts)
+print(f"Written {len(pkts)} packets to {out}")
+print()
+print("Expected ghost nodes after upload:")
+print(f"  {GHOST_PING1}  → no-response, icmp-unreachable")
+print(f"  {GHOST_PING2}  → no-response, icmp-unreachable")
+print(f"  {GHOST_ARP}  → no-response, arp-no-reply")
+print(f"  {HOP}  → ttl-exceeded")
+print()
+print(f"Real nodes (should NOT be tagged as ghost):")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)